ZipBuilder/Sources/ZipBuilder/FrameworkBuilder.swift

```diff
@@ -274,78 +294,25 @@ struct FrameworkBuilder {
       thinArchives.append(thinArchive)
     }
 
-    // Create the framework directory in the filesystem for the thin archives to go.
-    let frameworkDir = outputDir.appendingPathComponent("\(framework).framework")
-    do {
-      try fileManager.createDirectory(at: frameworkDir, withIntermediateDirectories: true)
-    } catch {
-      fatalError("Could not create framework directory while building framework \(framework). " +
-        "\(error)")
+    let frameworkDir = outputDir.appendingPathComponent("\(framework).xcframework")
+
+    let inputArgs = thinArchives.flatMap { url -> [String] in
+        return ["-framework", url.path]
     }
 
-    // Build the fat archive using the `lipo` command. We need the full archive path and the list of
-    // thin paths (as Strings, not URLs).
-    let thinPaths = thinArchives.map { $0.path }
-    let fatArchive = frameworkDir.appendingPathComponent(framework)
-    let result = syncExec(command: "/usr/bin/lipo", args: ["-create", "-output", fatArchive.path] + thinPaths)
+    print("About to create xcframework for \(frameworkDir.path) with \(inputArgs)")
+
+    // xcframework doesn't support legacy architectures: armv7, i386.
```

Comment:
Interesting. Definitely a case for us to drop 32 bit support sooner ...

Reply:
Investigating that one. It's woefully under-documented, but Twitter folks say it is possible.

Reply:
Was wrong about that. It does, just a bit tricky and under-documented.